pkg/notifiers/model.go

```diff
@@ -0,0 +1,6 @@
+package notifiers
+
+type Message struct {
```

Comment:
any reason why we have struct definition in its own file? Is the struct being referenced cross packages?